package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.112",
   "description": "The Cisco Spark for React library allows developers to easily incorporate Spark functionality into an application.",
   "scripts": {
-    "build": "npm run build:packages && npm run build:bundle",
-    "build:packages": "cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/build-packages.js",
-    "build:package": "cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/build-package.js",
-    "build:bundle": "cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/build-bundle.js",
-    "build:packagejson": "cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/update-package-json.js",
-    "build:widgets": "cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/build-widgets.js",
-    "clean": "npm run clean:snapshots && npm run clean:dist",
+    "build": "cross-env NODE_ENV=${NODE_ENV:-production} ./scripts/build/index.js",
+    "build:all": "npm run build all",
+    "build:package": "npm run build dist",
```

Comment:
interesting. I think I've found `--` isn't needed in basic use cases, but if you start throwing switches in for your own scripts (or possibly multiple params), `npm run` gets really confused.